packages/x-date-pickers/src/tests/describeValue/testShortcuts.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { expectPickerChangeHandlerValue } from 'test/utils/pickers/assertions';
+import { userEvent } from '@mui/monorepo/test/utils';
+import { chipClasses } from '@mui/material/Chip';
+import { pickersLayoutClasses } from '@mui/x-date-pickers/PickersLayout';
+import { DescribeValueTestSuite } from './describeValue.types';
+
+export const testShortcuts: DescribeValueTestSuite<any, 'picker'> = (ElementToTest, options) => {
+  const {
+    componentFamily,
+    render,
+    renderWithProps,
+    values,
+    emptyValue,
+    setNewValue,
+    ...pickerParams
+  } = options;
+
+  if (componentFamily !== 'picker') {
+    return;
+  }
+
+  describe('Picker shortcuts', () => {
+    it('should call onClose, onChange and onAccept when picking a shortcut without explicit changeImportance', () => {
+      const onChange = spy();
+      const onAccept = spy();
+      const onClose = spy();
+
+      render(
+        <ElementToTest
+          onChange={onChange}
+          onAccept={onAccept}
+          onClose={onClose}
+          defaultValue={values[0]}
+          open
+          closeOnSelect
+          slotProps={{
+            shortcuts: {
+              items: [
+                {
+                  label: 'Test shortcut',
+                  getValue: () => values[1],
+                },
+              ],
+            },
+          }}
+        />,
+      );
+
+      const shortcut = document.querySelector(
+        `.${pickersLayoutClasses.shortcuts} .${chipClasses.root}`,
+      )!;
```

Comment:
Nitpick, consider replacing with :
```suggestion
      const shortcut = screen.getByRole('button', { name: 'Test shortcut' });
```

Reply:
That's a lot better thanks!